### PR TITLE
moving codecov bash executor to their github

### PR DESCRIPTION
**Pull request by chayim**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -7,15 +7,15 @@ jobs:
   build:
     docker:
       - image: circleci/openjdk:11.0.7-jdk-buster
-      
+
       - image: circleci/mysql:8.0.4
         environment:
           MYSQL_ROOT_PASSWORD : my-secret-pw
-          MYSQL_ROOT_HOST: %          
+          MYSQL_ROOT_HOST: localhost
```
Comment (gkorland) [inline .circleci/config.yml:14]: @chayim how is this change related to this PR?

Comment (chayim) [reply .circleci/config.yml:14]: It's not - there are (several) issues with the mysql connection, and I accidentally added this in by merging my branch down. Removed + pushed back. Now it's just as broken as before - but without the supply issue. I'll open another PR to clean that.
